src/Microsoft.DotNet.Darc/src/Darc/Operations/GetDependencyGraphOperation.cs

```diff
@@ -174,6 +178,37 @@ private void LogDependencyGraph(DependencyGraph graph)
             LogIncoherencies(graph);
         }
 
+        private string GetSimpleRepoName(string repoUri)
+        {
+            int lastSlash = repoUri.LastIndexOf("/");
+            if ((lastSlash != -1) && (lastSlash < (repoUri.Length - 1)))
+            {
+                return repoUri.Substring(lastSlash + 1);
+            }
+            return repoUri;
+        }
+
+        private string GetGraphVizNodeName(DependencyGraphNode node)
+        {
+            return GetSimpleRepoName(node.RepoUri).Replace("-", "") + node.Commit;
+        }
+
+        private void LogGraphViz(DependencyGraph graph)
+        {
+            Console.WriteLine("digraph repositoryGraph {");
```

Comment:
what's "digraph"?

Reply:
the opening marker for the graphviz description.